.gitignore

```diff
@@ -23,3 +23,6 @@
 /.node_modules.ember-try/
 /bower.json.ember-try
 /package.json.ember-try
+
+# IDE stuff
+/.idea/
```

Comment:
There's obviously no harm in having this here but it's perhaps a bit sloppy to scatter ad hoc IDE stuff into the gitignore. Not a blocker, but this seems like it would better be solved by `.git/info/exclude`